docs/specs/supporting-tools.md

```diff
@@ -0,0 +1,68 @@
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## Dev container CLI
+
+There will be a dev container command line interface (CLI) that can take a `devcontainer.json` and create and configure a dev container from it.
+
+The publishing of this CLI is being discussed in an [issue](https://github.com/microsoft/dev-container-spec/issues/9).
+
+## GitHub Codespaces
+
+A [codespace](https://docs.github.com/en/codespaces/overview) is a development environment that's hosted in the cloud. Codespaces run on a variety of VM-based compute options hosted by GitHub.com, which you can configure from 2 core machines up to 32 core machines. You can connect to your codespaces from the browser or locally using Visual Studio Code.
+
+> **Tip:** If you've already built a codespace and connected to it, be sure to run **Codespaces: Rebuild Container** from the Command Palette (`kbstyle(F1)`) to pick up any changes you make.
+
+### Product specific properties
+GitHub Codespaces works with a growing number of tools and, where applicable, their `devcontainer.json` properties. For example, connecting the Codespaces web editor or VS Code enables the use of [VS Code properties](#visual-studio-code-remote---containers).
+
+### Product specific limitations
+
+Some properties may apply differently to Codespaces.
+
+| Property or variable | Type | Description |
+|----------|---------|----------------------|
+| `mounts` | array | Codespaces ignores "bind" mounts with the exception of the Docker socket. Volume mounts are still allowed.|
+| `workspaceMount` | string | Not yet supported in Codespaces. |
+| `workspaceFolder` | string | Not yet supported in Codespaces. |
+| `forwardPorts` | array | Codespaces does not yet support the `"host:port"` variation of this property. |
+| `portsAttributes` | object | Codespaces does not yet support the `"host:port"` variation of this property.|
+| `shutdownAction` | enum | Does not apply to Codespaces. |
+| `${localEnv:VARIABLE_NAME}` | Any | For Codespaces, the host is in the cloud rather than your local machine.|
+
+## Visual Studio Code Remote - Containers
+
+The [**Visual Studio Code Remote - Containers** extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) lets you use a [Docker container](https://docker.com) as a full-featured development environment. It allows you to open any folder inside (or mounted into) a container and take advantage of Visual Studio Code's full feature set. There is more information in the Remote - Containers [documentation](https://code.visualstudio.com/docs/remote/containers).
```

Comment:
Might want to remove "Docker container" here and just use "container" considering we're trying to be open on engine, etc with the spec.

Reply:
Makes sense - updated just to "container" without a hyperlink.